### PR TITLE
project-news-site J

### PR DESCRIPTION
**Pull request by Jsfrulle**
https://jsfrulle-movie-night.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+          <a id="nav-button-3" href="https://www.aftonbladet.se/tipsa" target="_blank" class="nawbar-btn">Kontakta oss</a>
+          <a id="nav-button-4" href="https://www.aftonbladet.se/nojesbladet/tv" target="_blank" class="nawbar-btn">Utforska</a>
+        </div>
+      </div>
+      </div>
+    
+    
+      <div class="container-header">
+
+      <header>
```
Comment (Kaliine) [inline index.html:48]: Just thinking of html semantics - perhaps consider to rename the `<header>` tag to something else, for example `<section>`. Or is there a reason you call this a header?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          Galopperande underhållande? Tveklöst. Netflix
+        </p> 
+      
+        <a class="stream-big-btn" target="blank" href="https://www.netflix.com/se/title/80996810"
+        >Streama ”Cocaine cowboys” här</a>
+      </section>
+      </div>
+
+
+      <div class="container-regular-footer">
```
Comment (Kaliine) [inline index.html:87]: Thinking of html semantics - is this a footer? Perhaps consider to rename the class?

File: style.css [hunk 3]
```diff
@@ -0,0 +1,735 @@
...
+
+@media (max-width: 355px) {
+  .navbar-items {
+    margin-left: -40%;
+  }
+  .nawbar-btn {
+    font-size: 75%;
+    margin: 1% 2% 0 2%;
+  }
+
```
Comment (Kaliine) [inline style.css:730]: How come you used 10 media queries?

**Review by Kaliine — COMMENTED** [review]
For me, as a complete beginner, it was inspiring to read your code. Your site has a fun design, a lot of interesting features and fulfills the general requirements and beyond! Great work!

**Comment by Jsfrulle** [comment]
Hi Kaliine,  I did change my header and "container-regular-footer" but cant push the changes :/ 
And the media queries is for the navbar to always be in the right spot (next time when I use the position:fix" I'm going to read what this type of position doing to the element, Like 65% of my work was to fix this navbar, haha, and if I would to this task again I would do a other soulotion - ten media queries is just for to fix the navbar!

Thanks for good and constructive feedback! :)

File: style.css (hunk 3 above)
Comment (Jsfrulle) [reply style.css:730]: Hi Kaliine, I did change my header and "container-regular-footer" but cant push the changes :/
And the media queries is for the navbar to always be in the right spot (next time when I use the position:fix" I'm going to read what this type of position doing to the element, Like 65% of my work was to fix this navbar, haha, and if I would to this task again I would do a other soulotion - ten media queries is just for to fix the navbar!

Thanks for good and constructive feedback! :)

**Pull request closed without merging**